tripal/src/Services/TripalPublish.php

```diff
@@ -476,10 +476,18 @@ protected function getEntityTitles($matches) {
           $delta = 0;
           $field = $this->field_info[$field_name]['instance'];
           $main_prop = $field->mainPropertyName();
-          $value = $match[$field_name][$delta][$main_prop]['value']->getValue();
-          if ($value !== NULL) {
-            $entity_title = trim(preg_replace("/\[$field_name\]/", $value,  $entity_title));
+          $value = '';
+          if (array_key_exists($delta, $match[$field_name])) {
+            $value = $match[$field_name][$delta][$main_prop]['value']->getValue();
           }
+          $value = '';
+          if (array_key_exists($delta, $match[$field_name])) {
+            $value = $match[$field_name][$delta][$main_prop]['value']->getValue();
+          }
```

Comment:
```suggestion
```
I'm not sure what happened here but the code was duplicated. This removes the second copy. I'll just go ahead and commit this...